ff/serving/api/api.go

```diff
@@ -196,7 +242,7 @@ func main() {
 			panic(fmt.Sprintf("health check HTTP server failed: %+v", err))
 		}
 	}()
-	serv, err := NewApiServer(logger, "0.0.0.0:7878", "sandbox-metadata-server:8080")
+	serv, err := NewApiServer(logger, "0.0.0.0:7878", "sandbox-metadata-server:8080", "sandbox-serving-server:8080")
```

Comment:
should port be derived from environmental variable in helm chart, or more globally to be seen higher? Just because 8080 is typed in plaintext in seperate places

Reply:
Was leaving this as part of the helm task to remove hardcoded addresses so I can do them all at once. There are other things in the chart that all need to change together

Reply:
ok